recipes/mold/all/test_package/conanfile.py

```diff
@@ -0,0 +1,11 @@
+import os
+from conans import ConanFile, tools
+from conan.tools.build import cross_building
+
+class TestPackageConan(ConanFile):
+    settings = "os", "arch"
```

Comment:
```suggestion
    settings = "os", "arch", "build_type", "compiler"
```
Keep full setting always. There is a Hook warning about it.